scipy/stats/_covariance.py

```diff
@@ -18,6 +18,9 @@ class Covariance:
     object representing a covariance matrix using any of several
     decompositions and perform calculations using a common interface.
 
+    Note that the `Covariance` class cannot be instantiated directly.
+    Instead, use one of the factory methods (e.g. `Covariance.from_diagonal`).
```

Comment:
I would make it more prominent with either a note or warning markup
```suggestion
    .. note::

        The `Covariance` class cannot be instantiated directly.
        Instead, use one of the factory methods (e.g. `Covariance.from_diagonal`).
```

Reply:
OK, we can. If the user doesn't notice, I think they will notice the error message that says the same thing, though.

Reply:
Done.